### PR TITLE
feat: add dev cluster to env options for jina operator integration tests

### PR DESCRIPTION
**Pull request by winstonww**

### CODE REVIEW

File: .github/workflows/operator-integration-tests-v2.yml [hunk 1]
```diff
@@ -12,12 +12,50 @@ on:
...
+        description: Choose the environment (i.e. which cluster to run the integration tests on)
+        required: true
+        options:
+          - CI
+          - dev
+
 
 concurrency:
   group: operator-e2e-tests
   cancel-in-progress: false
```
Comment (deepankarm) [inline .github/workflows/operator-integration-tests-v2.yml:26]: Currently, concurrency is grouped by string `operator-e2e-tests`. This makes sure only one job is running at a time. Now that we have 2 envs, let's change group to consider `env` as well.

File: .github/workflows/operator-integration-tests-v2.yml [hunk 2]
```diff
@@ -12,12 +12,50 @@ on:
...
+      - id: output-host
+        run: |
+          if [[ ${{ github.event.inputs.env }} == 'CI' ]]; then
+            echo "host=api-ci-episr.wolf.jina.ai" >> $GITHUB_OUTPUT
+          else
+            echo "host=api-dev-lpzje.wolf.jina.ai" >> $GITHUB_OUTPUT
+          fi
+
+
   notify-user:
```
Comment (deepankarm) [inline .github/workflows/operator-integration-tests-v2.yml:59]: Can we also include the `env` in `notify-user` text?

File: .github/workflows/operator-integration-tests-v2.yml (hunk 1 above)
Comment (winstonww) [reply .github/workflows/operator-integration-tests-v2.yml:26]: Addressed.

File: .github/workflows/operator-integration-tests-v2.yml [hunk 3]
```diff
@@ -351,11 +389,13 @@ jobs:
     needs: 
       - operator-tests
       - api-tests
+      - set-env
     if: ${{ (github.event.inputs.branch != 'main') && (github.event.inputs.only_api_changed == 'false') }}
-    uses: jina-ai/cloud-ops/.github/workflows/operator-backward-compatibility-tests.yml@master
+    uses: jina-ai/cloud-ops/.github/workflows/operator-backward-compatibility-tests.yml@feat-add-dev-operator-integration-tests
```
Comment (zac-li) [inline .github/workflows/operator-integration-tests-v2.yml:394]: should revert this?

**Review by zac-li — APPROVED** [review]

File: .github/workflows/operator-integration-tests-v2.yml (hunk 3 above)
Comment (winstonww) [reply .github/workflows/operator-integration-tests-v2.yml]: Reverted, thanks.

File: .github/workflows/operator-integration-tests-v2.yml [hunk 4]
```diff
@@ -259,14 +297,14 @@ jobs:
           status: ${{ job.status }}
           fields: eventName,job
           text: |
-            :no_entry: `${{ matrix.test-path }}` failed for branch `${{ github.event.inputs.branch }}`
+            :no_entry: `${{ matrix.test-path }}` failed for branch `${{ github.event.inputs.branch }} (cluster: ${{ github.event.inputs.env }})`
```
[the remark this answers is not in the document]
Comment (winstonww) [reply .github/workflows/operator-integration-tests-v2.yml]: Addressed, thanks

**Pull request merged**